src/bindings/python/src/pyopenvino/graph/dict_attribute_visitor.hpp

```diff
@@ -84,12 +84,16 @@ class DictAttributeSerializer : public ov::AttributeVisitor {
     void on_adapter(const std::string& name, ov::ValueAccessor<std::vector<float>>& adapter) override;
     void on_adapter(const std::string& name, ov::ValueAccessor<std::vector<double>>& adapter) override;
 
+    bool contains_attribute(const std::string& name) {
+        if (m_attributes.contains(name)) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
     template <typename T>
     T get_attribute(const std::string& name) {
-        NGRAPH_CHECK(m_attributes.contains(name),
```

Comment:
Why removed? Can you explain what will happen when name is not in `m_attributes`.

Reply:
It will simply throw KeyError. So it virtually does the same thing, checks if the "attribute" is there.

I don't quite get the idea of leaving old `NGRAPH_CHECK`. It will just run `contains_attribute` once more. I can replace it with try-catch and change it. However our use-cases of this serializer-dict are limited and actually only place that use it is Node class... which I edited to control flow of `contains_attribute` first and only then running `get_attribute`.

So I just removed it to not double-check something that is already done.

Reply:
If it should be kept, should be replaced to `OPENVINO_ASSERT`

Reply:
@jiwaszki If you are sure about this change then is Ok form me.